src/assets/pages/blogposts/post-48.html

```diff
@@ -0,0 +1,33 @@
+<p><i>Em cerca de um mês o número de municípios integrados foi duplicado, superando todas as expectativas. </i></p>
```

Comment:
```suggestion
<p><em>Em cerca de um mês o número de municípios integrados foi duplicado, superando todas as expectativas. </em></p>
```

De todos os `<em>` que tinha antes, este estava certo. Aqui, enquanto linha fina, realmente é uma "enfase" e não um mero "itálico"